src/core/src/lib/extensions/core/core.ts

```diff
@@ -29,6 +29,32 @@ export class CoreExtension implements FormlyExtension {
       });
     }
 
+    Object.defineProperty(field, 'get', {
+      value: (key) => {
+        let formlyField: FormlyFieldConfig = null;
+
+        const findField = (f: FormlyFieldConfig, keyValue: string | number) => {
```

Comment:
move this to [util](https://github.com/ngx-formly/ngx-formly/blob/main/src/core/src/lib/utils.ts) file.